### ISSUE TITLE
an update to Spigot 1.17 is needed :)

### ISSUE THREAD

**Issue by CloudeLecaw** [EricLangezaal/PetDragon#5]
```
[15:46:24] [Server thread/INFO]: [PetDragon] Enabling PetDragon v1.4.2
[15:46:24] [Server thread/INFO]: [21m[0;31;1m[PetDragon] Unsupported minecraft version! Check the download page for supported versions![m
[15:46:24] [Server thread/INFO]: [21m[0;31;1m[PetDragon] Plugin will disable to prevent crashing![m
```

**Comment by EricLangezaal** [comment]
I am well aware that it doesn't work on 1.17 yet haha, as you know I basically need to recode the whole thing for it:). I will 100% update this plugin, but it will take quite some more time than most plugins, so it could take a week or two perhaps. 
Another thing is that I was halfway through a massive update, but unfortunately didn't have time to finish it, so will have to stash parts of my unfinished update somewhere.

**Comment by nfsprodriver** [comment]
I made a build for 1.17 only, but the dragon rises instantly to y 4000 and kills me haha. The minecraft decompiler from spiot is a bit tricky in keeping function names. So I leave it to the pros, unless you're interested ;)

**Comment by EricLangezaal** [comment]
Haha that might not be the desired behaviour. Yeah 1.17 is quite difficult, especially given that Spigot decided to completely change packaging, breaking PetDragon:). So I will have to switch to a multi-module project probably. I already have the 1.17 version mapped, just need to create separate modules etc.

I also added a ton of changes (some of which I had to stash for the future) and basically rewrote the entire dragon’s tick for performance and configurability, so translating that for all versions will be fun too.
